src/python/test/test_dxclient.py

```diff
@@ -359,6 +359,28 @@ def test_dx_shell(self):
         shell.sendline("echo find projects | dx sh")
         shell.expect("project-")
 
+    def test_dx_watch_invalid_auth(self):
```

Comment:
one thing I'm not sure of.. do I need to put some sort of skip on this test? I'm attempting to make sure that the environment is the same after this test as it was before, so I don't see why we should skip it - but maybe I'm missing something?

Reply:
It looks that nothing needs to be skipped here. Usually we skip job runs or tests that create some global objects, orgs, etc. and only run them in special test suite environments.